cluster-provision/k8s/1.17/provision.sh

```diff
@@ -21,6 +21,9 @@ yum -y remove firewalld
 # Required for iscsi demo to work.
 yum -y install iscsi-initiator-utils
 
+# To prevent preflight issue realted to tc not found
+dnf install -y tc
```

Comment:
Is that a new requirement?

Reply:
I found a warning at provisiong from kubeadm missing `tc`